src/misc.cpp

```diff
@@ -149,13 +146,16 @@ string engine_info(bool to_uci) {
   string month, day, year;
   stringstream ss, date(__DATE__); // From compiler, format is "Sep 21 2008"
 
-  ss << "Stockfish " << Version << setfill('0');
+  /// Version number. If Version is left empty, then compile date in the format
+  /// DDMMYY and show in engine_info.
+  ss << "Stockfish " << version << " " << setfill('0');
 
-  if (Version.empty())
```

Comment:
I think we should probably only add date and sha if we have 'dev' in the version string. Otherwise it is unique anyway, and rather keep official releases 'simple'.